src/Data/useDataQuery.ts

```diff
@@ -77,12 +81,21 @@ export const useDataQuery = <T extends GenericRawData>(
 
   const key = dataKey.replace(YEAR_PLACEHOLDER, year || '')
 
-  const url = metadata && year ? metadata[key].csvURL : ''
+  let url = ''
+
+  if (year === '2022') {
```

Comment:
This appears to only get the retrieve the CSV from the local data folder for 2022 data -- doesn't go to the Catalogue.

Would it be possible to check the Data Catalogue first, then go to the local data folder if it isn't in the Data Catalogue? Otherwise, it appears this will need to be updated once the data is in the Data Catalogue.

Reply:
Yes, that's correct. My thinking here, which I'm open to revising, is roughly:

* The advantages of having a local fallback:
    * If the Data Catalogue is inaccessible, the data can still be browsed.
    * No future code update or deployment is required once the 2022 data goes live on the Data Catalogue.

* The disadvantages:
    * If we ever update the Data Catalogue with new data, we'll also have to update the local CSV files in order to keep the data consistent. This would also require a redeployment of the application. This is a (minor) potential maintenance burden.

Personally, I come down on the side of making the necessary code adjustment in the near future to only ever load data from the Data Catalogue, without a local fallback. But I can see the advantages of having a local fallback, too —  especially if we're quite certain the data never needs updating. I will check in with Amelia to get a sense of how often the data has been updated in the past.